components/main_page/wikis/leagueoflegends/main_page_layout_data.lua

```diff
@@ -178,7 +178,7 @@ return {
 			count = {
 				method = 'LPDB',
 				table = 'datapoint',
-				conditions = '[[type::hero]]',
+				conditions = '[[type::character]]',
```

Comment:
we likely want this temporarily
```suggestion
				conditions = '[[type::character]] OR [[type::hero]]',
```
reason:
with changing the infobox the data in lpdb is not insta changed on all pages
the job queue will likely take care of the most over some minutes, but it might miss a few
(even in the good case the main page would be "broken" several minutes)

hence imho do this here temporarily and open a follow up PR after merge to remove the old type in the query condition again (to be merged after all lpdb datas have been updated accordingly)

Reply:
Manual mass purge, easy enough, number will be wrong for a min basically